server.js

```diff
@@ -21,9 +21,73 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Hello!");
 });
 
+
+app.get("/bookshelf/page/:page", (req, res) => {
+  const page = req.params.page;
+  console.log(`requested page ${page}`);
+  const limit = 10
```

Comment:
I really like this that you added a limit to this and also worked with pagination.